src/status_im/utils/ethereum/abi_spec.cljs

```diff
@@ -41,7 +41,7 @@
 
 (defn hex-to-number [x]
   (when x
-    (.toNumber (dependencies/Web3.prototype.toBigNumber (str "0x" x) 16))))
+    (dependencies/Web3.prototype.toBigNumber (str "0x" x) 16)))
```

Comment:
`.toBigNumber` has only one parameter, so 16 is just confusing and better to remove it
how does it work without `.toNumber`?